amplipi/streams.py

```diff
@@ -1104,13 +1104,13 @@ def disconnect(self):
   def info(self) -> models.SourceInfo:
     # Opens and reads the metadata.json file every time the info def is called
     # uses a file lock so that the file cannot be read while writing and vice/versa. If you let those processes run into eachother, there are errors
-    try:
-      meta_read = open(f"lms_{str(self.name).replace(' ', '_')}_metadata.json", "r", encoding="utf-8")
-      fcntl.flock(meta_read, fcntl.LOCK_EX)
-      self.meta = json.loads(meta_read.read())
-    finally:
-      fcntl.flock(meta_read, fcntl.LOCK_UN)
-      meta_read.close()
+    with open(f"lms_{str(self.name).replace(' ', '_')}_metadata.json", "r", encoding="utf-8") as meta_read:
```

Comment:
This does not handle any file i/o errors. I would expect an i/o error to be suppressed in this method and for the info to be returned to be the default lms info or something about being unable to connect to LMS server.